routes/web.php

```diff
@@ -13,6 +15,9 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::redirect('/', '/login');
+
+Route::view('/register', 'auth.register')->name('user.register');
```

Comment:
for leveraging your involvement in laravel. Please take reference of `laravel-ui` package as well